generation/remap.rsp

```diff
@@ -3223,6 +3225,7 @@ __AnonymousBase_WinUser_L1173_C46=Base
 __AnonymousBase_WinUser_L13554_C43=Base
 __AnonymousBase_WinUser_L13558_C43=Base
 __AnonymousEnum_d2d1_L89_C1=D2D1_INTERPOLATION_MODE_DEFINITION
+__AnonymousRecord_roapi_L45_C9*=IntPtr
```

Comment:
Why does this need to be an `IntPtr`?

Reply:
The definition for this struct in `roapi.h` is 
```cpp
typedef struct {} *RO_REGISTRATION_COOKIE;
```